tests/daemon/test-machine-id-provider.c

```diff
@@ -31,7 +31,9 @@
 #define HYPHENS_IN_ID 4
 
 #define TESTING_FILE_PATH "/tmp/testing-machine-id"
+#define TESTING_OVERRIDE_FILE_PATH "/tmp/override-testing-machine-id"
```

Comment:
I appreciate the existing test does the same but I don't really like using fixed paths in `/tmp` here. I think the fixture setup should create the temporary files.